install.sh

```diff
@@ -32,50 +32,52 @@ mv vc/tools/MSVC vc/tools/msvc
 # /DEFAULTLIB:"OLDNAMES", which lld-link doesn't find on a case sensitive
 # filesystem. Therefore add matching case symlinks for this, to allow
 # linking MSVC built objects with lld-link.
-cd $(echo vc/tools/msvc/* | awk '{print $1}')/lib
+cd "$(echo vc/tools/msvc/* | awk '{print $1}')/lib"
 for arch in x86 x64 arm arm64; do
-    cd $arch
+    cd "$arch"
```

Comment:
The change on this line is kinda superfluous IMO (I know you did it for consistency though), as `$arch` only is either of the 4 single words listed above.

Reply:
True. Again my thinking was better to have unnecessary quotes than forgetting them somewhere. :smile: